docs/architecture/011-capi-elements-as-components.md

```diff
@@ -0,0 +1,41 @@
+# Elements as Components
+
+## Context
+
+CAPI provides a representation of content which is strongly typed and like components.
+
+## Decision
+
+### CAPI Elements as Components
+
+The CAPI Blocks API exposes a suite of strongly typed elements. A component should be made for each element to render it.
+
+#### Elements Types
+
+Each element must have a type in dotcom rendering which specifies the data it may contain. It must have a `_type` field with a string literal of the elements name. This is to be used identifying which component to render.
```

Comment:
It must have a `_type` field with a string literal representing the element's name. This should be used to identify which component to render.